src/Controls/tests/DeviceTests/Elements/Shell/ShellTests.Windows.cs

```diff
@@ -519,6 +519,64 @@ public async Task ShellToolbarWithOnlyToolbarItemsIsVisible()
 			});
 		}
 
+		[Fact(DisplayName = "Shell Add then Remove Items")]
+		public async Task ShellAddRemoveItems()
```

Comment:
Can we move this to `ShellTests` so it runs on all platforms? Nothing here is specific to `Windows` so I think it'll add some coverage on `iOS` and `Android` as well. 

I tested on `Android` and it passes. I feel good about iOS :-)

Reply:
Moved this + addressed above feedback :)